cmd/natdisco/main.go

```diff
@@ -22,40 +22,43 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"strings"
 	"time"
 
 	"github.com/mysteriumnetwork/node/nat/behavior"
 )
 
+const addressListSeparator = ";"
+
 var (
-	address      = flag.String("server", "stun.stunprotocol.org:3478", "STUN server address")
-	reqTimeout   = flag.Duration("req-timeout", 3*time.Second, "timeout to wait for each STUN server response")
-	totalTimeout = flag.Duration("total-timeout", 15*time.Second, "overall operation deadline")
+	addressList  = flag.String("servers", "stun.mysterium.network:3478;stun.stunprotocol.org:3478;stun.sip.us:3478", "semicolon-separated list of STUN servers")
```

Comment:
I think we are using comma-separated lists in other places, it will be good to keep it consistent. 
```suggestion
	addressList  = flag.String("servers", "stun.mysterium.network:3478;stun.stunprotocol.org:3478;stun.sip.us:3478", "semicolon-separated list of STUN servers")
```

Reply:
Updated